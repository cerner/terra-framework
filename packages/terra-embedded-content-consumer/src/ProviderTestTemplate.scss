.embedded-content-provider-test-template-body {
  height: auto !important;
  width: 100%;
}
