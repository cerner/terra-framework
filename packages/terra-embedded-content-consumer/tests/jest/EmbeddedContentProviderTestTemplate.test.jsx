import React from 'react';
import ProviderTestTemplate from '../../lib/EmbeddedContentProviderTestTemplate';

// Snapshot Tests
it('should render the provider test template', () => {
  const providerExample = (
    <ProviderTestTemplate>
      <div>Content to display</div>
    </ProviderTestTemplate>
  );

  const wrapper = render(providerExample);
  expect(wrapper).toMatchSnapshot();
});

it('should render the provider test template with a custom class name', () => {
  const providerExample = (
    <ProviderTestTemplate className="container">
      <div>Content to display</div>
    </ProviderTestTemplate>
  );

  const wrapper = render(providerExample);
  expect(wrapper).toMatchSnapshot();
});
