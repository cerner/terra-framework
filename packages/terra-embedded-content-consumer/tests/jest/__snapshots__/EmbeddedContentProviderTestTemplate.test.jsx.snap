// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the provider test template 1`] = `
<div
  data-embedded-content-consumer-provider-test-template="true"
>
  <div>
    Content to display
  </div>
</div>
`;

exports[`should render the provider test template with a custom class name 1`] = `
<div
  class="container"
  data-embedded-content-consumer-provider-test-template="true"
>
  <div>
    Content to display
  </div>
</div>
`;
